ui/raidboss/data/05-shb/dungeon/akadaemia_anyder.js

```diff
@@ -61,15 +61,15 @@
             en: 'Tank Buster on YOU',
             de: 'Tankbuster auf DIR',
             fr: 'Tankbuster sur VOUS',
-            ko: '나에게 탱크버스터',
+            ko: '탱버 대상자',
           };
         }
         if (data.role == 'healer') {
           return {
             en: 'Buster on ' + data.ShortName(matches.target),
             de: 'Tankbuster auf ' + data.ShortName(matches.target),
             fr: 'Tankbuster sur ' + data.ShortName(matches.target),
-            ko: data.ShortName(matches.target) + '에게 탱크버스터',
+            ko: '"' + data.ShortName(matches.target) + '"탱버',
```

Comment:
Seeing these changes here makes me think that it'd be really nice to switch everything to using Responses.tankbuster() to be consistent and not have to duplicate this everywhere.